StripeIdentity/StripeIdentityTests/Unit/NativeComponents/ViewControllers/DocumentCaptureViewControllerTest.swift

```diff
@@ -38,7 +37,7 @@ final class DocumentCaptureViewControllerTest: XCTestCase {
 
     let mockError = NSError(domain: "mock_error", code: 100, userInfo: nil)
 
-    override class func setUp() {
+    override static func setUp() {
```

Comment:
Curious-- did changing this from `class` to `static` change anything?

Reply:
Doesn't change anything here, the linter made me